presto-native-execution/src/test/java/com/facebook/presto/nativeworker/PrestoNativeQueryRunnerUtils.java

```diff
@@ -140,6 +147,105 @@ public static QueryRunner createJavaQueryRunner(Optional<Path> baseDataDirectory
         return queryRunner;
     }
 
+    public static QueryRunner createJavaIcebergQueryRunner(boolean addStorageFormatToPath)
+            throws Exception
+    {
+        return createJavaIcebergQueryRunner(Optional.of(getNativeQueryRunnerParameters().dataDirectory), ICEBERG_DEFAULT_STORAGE_FORMAT, addStorageFormatToPath);
+    }
+
+    public static QueryRunner createJavaIcebergQueryRunner(String storageFormat)
+            throws Exception
+    {
+        return createJavaIcebergQueryRunner(Optional.of(getNativeQueryRunnerParameters().dataDirectory), storageFormat, true);
+    }
+
+    public static QueryRunner createJavaIcebergQueryRunner(Optional<Path> baseDataDirectory, String storageFormat, boolean addStorageFormatToPath)
+            throws Exception
+    {
+        ImmutableMap.Builder<String, String> icebergPropertiesBuilder = new ImmutableMap.Builder<>();
+        icebergPropertiesBuilder
+                .put("hive.pushdown-filter-enabled", "true")
+                .put("hive.parquet.writer.version", "PARQUET_1_0");
+
+        Optional<Path> dataDirectory = addStorageFormatToPath ? baseDataDirectory.map(path -> Paths.get(path.toString() + '/' + storageFormat)) : baseDataDirectory;
+
+        DistributedQueryRunner queryRunner = createIcebergQueryRunner(
+                ImmutableMap.of(
+                        "parse-decimal-literals-as-double", "true",
+                        "regex-library", "RE2J",
+                        "offset-clause-enabled", "true",
+                        "query.max-stage-count", "110"),
+                icebergPropertiesBuilder.build(),
+                FileFormat.valueOf(storageFormat),
+                false,
+                false,
+                OptionalInt.empty(),
+                Optional.empty(),
+                dataDirectory);
+
+        return queryRunner;
+    }
+
+    public static QueryRunner createNativeIcebergQueryRunner(
```

Comment:
Can you please move this methods to be the last of the native series? It's being called by the below overload methods